### DISCUSSION TITLE
A question about SSH port

### DISCUSSION THREAD

**Discussion by hack3rcon** [fail2ban/fail2ban#3695] [Q&A] [answered]
Hello,
Fail2ban uses `port = ssh` by default. If I change the SSH port from 22 to another number, do I have to change the port value to that number as well?

Thank you.

**Comment by sebres** [comment, marked as answer]
It depends:
- if you changed it in `/etc/services` then very probably not, since ssh will map to different port.
- if you changed it in `/etc/sshd_config` only, then definitely yes, since ssh will still map to 22.

A restart of jail or fail2ban is expected in any case (like a restart of sshd).
